go/src/koding/go-webserver/envdata.go

```diff
@@ -17,10 +17,10 @@ import (
 const myPermissionsAndRolesPath = "/-/my/permissionsAndRoles"
 
 var defClient = httputil.NewClient(&httputil.ClientConfig{
-	DialTimeout:           1 * time.Second,
-	RoundTripTimeout:      2 * time.Second,
-	TLSHandshakeTimeout:   2 * time.Second,
-	ResponseHeaderTimeout: 2 * time.Second,
+	DialTimeout:           30 * time.Second,
```

Comment:
yes, the workers requsting this endpoint recieves requests from elb, if the
parent request is terminated, spans should be terminated too as soon as
possible
On Fri, 5 Aug 2016 at 17:52, Sönmez Kartal <notifications@github.com> wrote:

> In go/src/koding/go-webserver/envdata.go
> <https://github.com/koding/koding/pull/8701#discussion_r73705086>:
>
> > @@ -17,10 +17,10 @@ import (
> >  const myPermissionsAndRolesPath = "/-/my/permissionsAndRoles"
> >
> >  var defClient = httputil.NewClient(&httputil.ClientConfig{
> > -	DialTimeout:           1 * time.Second,
> > -	RoundTripTimeout:      2 * time.Second,
> > -	TLSHandshakeTimeout:   2 * time.Second,
> > -	ResponseHeaderTimeout: 2 * time.Second,
> > +	DialTimeout:           30 * time.Second,
>
> This client makes requests to localhost directly.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/koding/koding/pull/8701/files/630f0711ffb64c6b144049b0606020bd83c8d2e3#r73705086>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA1sLPceNb1R5yNV8-mgUs1m_ozDSalbks5qc04pgaJpZM4JbvOL>
> .
>
-- 
Cihangir.
Software Engineer @ koding.com